Index: API_B2B_Center/test/test_migrated_document.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/API_B2B_Center/test/test_migrated_document.py b/api_b2b_center/test/test_migrated_document.py
rename from API_B2B_Center/test/test_migrated_document.py
rename to api_b2b_center/test/test_migrated_document.py
--- a/API_B2B_Center/test/test_migrated_document.py	
+++ b/api_b2b_center/test/test_migrated_document.py	
@@ -1,7 +1,7 @@
 import allure
 
-from API_B2B_Center.endpoints.migrated_document import MigratedDocument
-from API_B2B_Center.test.data import EXPECTED_RESPONSE_BODY_MIGRATE_CHILD_DOCUMENT, \
+from api_b2b_center.endpoints.migrated_document import MigratedDocument
+from api_b2b_center.test.data import EXPECTED_RESPONSE_BODY_MIGRATE_CHILD_DOCUMENT, \
     EXPECTED_RESPONSE_BODY_DOCUMENT_ALREADY_EXISTS, EXPECTED_RESPONSE_BODY_DOCUMENT_ALREADY_EXISTS_PAYLOAD, \
     EXPECTED_RESPONSE_BODY_MIGRATE_CHILD_DOCUMENT_PAYLOAD, EXPECTED_RESPONSE_BODY_SUPPLIER_NOT_FOUND_PAYLOAD, \
     EXPECTED_RESPONSE_BODY_SUPPLIER_NOT_FOUND, EXPECTED_RESPONSE_BODY_CONTRACT_KIND_CODE_REQUIRED_PAYLOAD, \
Index: API_B2B_Center/endpoints/migrated_document.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/API_B2B_Center/endpoints/migrated_document.py b/api_b2b_center/endpoints/migrated_document.py
rename from API_B2B_Center/endpoints/migrated_document.py
rename to api_b2b_center/endpoints/migrated_document.py
--- a/API_B2B_Center/endpoints/migrated_document.py	
+++ b/api_b2b_center/endpoints/migrated_document.py	
@@ -2,7 +2,7 @@
 import requests
 from dotenv import load_dotenv
 import allure
-from API_B2B_Center.test.data import URL
+from api_b2b_center.test.data import URL
 
 # Загрузка переменных окружения из .env файла
 load_dotenv()
