Index: .github/workflows/python-tests.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>name: Run Tests\r\n\r\non:\r\n  workflow_dispatch: # Позволяет запускать рабочий процесс вручную\r\n\r\njobs:\r\n  test:\r\n    runs-on: ubuntu-latest # Выбор операционной системы для выполнения тестов\r\n\r\n    steps:\r\n    - name: Checkout repository\r\n      uses: actions/checkout@v2 # Клонируем репозиторий\r\n\r\n    - name: Set up Python\r\n      uses: actions/setup-python@v2 # Устанавливаем Python\r\n      with:\r\n        python-version: '3.8' # Замените на нужную версию Python\r\n\r\n    - name: Install dependencies\r\n      run: |\r\n        python -m pip install --upgrade pip\r\n        pip install -r requirements.txt # Убедитесь, что в корне вашего репозитория есть requirements.txt\r\n\r\n    - name: Run tests\r\n      run: |\r\n        pytest # Замените на команду, которую вы используете для запуска тестов
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.github/workflows/python-tests.yml b/.github/workflows/python-tests.yml
--- a/.github/workflows/python-tests.yml	
+++ b/.github/workflows/python-tests.yml	
@@ -1,8 +1,7 @@
 name: Run Tests
 
 on:
-  workflow_dispatch: # Позволяет запускать рабочий процесс вручную
-
+  workflow_dispatch:
 jobs:
   test:
     runs-on: ubuntu-latest # Выбор операционной системы для выполнения тестов
@@ -14,7 +13,7 @@
     - name: Set up Python
       uses: actions/setup-python@v2 # Устанавливаем Python
       with:
-        python-version: '3.8' # Замените на нужную версию Python
+        python-version: '3.12'
 
     - name: Install dependencies
       run: |
